dataobj/schedule_entry.h

```diff
@@ -47,7 +48,10 @@ struct schedule_entry_t
 		UNLOAD_ALL        = 1U << 5, // The convoy unloads all loads here.
 		LOAD_BEFORE_DEP   = 1U << 6, // The convoy loads just before the departure.
 		TRANSFER_INTERVAL = 1U << 7,
-	};
+		REVERSE_CONVOY	= 1U << 0,// The reverse order of vehicles in convoy
+		REVERSE_p_c		= 1U << 1,// The reverse order of connected convoys
```

Comment:
[Q] Why are these two flags separately? Should the parent-child order always be reversed when the vehicle order in the convoy is reversed?

Reply:
The reversinig convoi and reversing parent-child order are independent.
For example, when the Locomotive coupling to other vehicles, the paprent-child relation should be decided with original way, but the Locomotive direction should be reversed.